workflows/differentialabundance.nf

```diff
@@ -38,11 +38,12 @@ if (params.study_type == 'affy_array'){
 if (params.control_features) { ch_control_features = file(params.control_features, checkIfExists: true) } else { ch_control_features = [[],[]] } 
 if (params.gsea_run) { 
     if (params.gsea_gene_sets){
-        gene_sets_file = file(params.gsea_gene_sets, checkIfExists: true)
+        gene_sets_files = params.gsea_gene_sets.split(" ")
+        gene_sets_files = gene_sets_files.collect { file(it, checkIfExists: true) }
     } else {
         error("GSEA activated but gene set file not specified!")
     }
-} else { gene_sets_file = [] } 
+} else { gene_sets_files = [] }
```

Comment:
```suggestion
} 
```
This variable isn't even used if the gene sets functionality is deactivated, so we can probably scrap it